lib/c-api/wasmer_wasm.h

```diff
@@ -29,6 +29,9 @@
 #  define DEPRECATED(message) __declspec(deprecated(message))
 #endif
 
+// The `jit` feature has been enabled for this build.
+#define WASMER_JIT_ENABLED
+
```

Comment:
?

Reply:
It was missing in the `wasmer_wasm.h` file. We should provide the `jit` feature.